assets/base.css

```diff
@@ -566,7 +566,7 @@ table:not([class]) th {
 }
 
 .light {
-  opacity: 0.7;
+  opacity: 0.75;
```

Comment:
the color applied to the body is already using a `.75` opacity. So I will change the approach as this is currently just a repetition of what was already there in the first place 🤔